Pipedrive extract source doc (#1146)
rudderlabs/rudderstack-docs: docs/sources/extract/pipedrive-v2.mdx

---
title: "Pipedrive V2"
description: Sync data from Pipedrive to your warehouse destination via RudderStack.
aliases: ["Pipedrive source"]
---

> [Pipedrive](https://www.pipedrive.com/) is a popular sales CRM and pipeline management tool that lets you manage your leads, track all your customer communications, automate administrative tasks, and much more.

Reviewer: ```suggestion
[Pipedrive](https://www.pipedrive.com/) is a popular sales CRM and pipeline management tool. It lets you manage your leads, track all your customer communications, and automate administrative tasks.
```

This document guides you in setting up Pipedrive as a source in RudderStack. Once configured, RudderStack automatically ingests your Pipedrive data and routes it to your specified data warehouse destination.

<div class="warningBlock">
All the Cloud Extract sources support sending data only to a <Link to="/destinations/warehouse-destinations/">data warehouse destination</Link>.
</div>

## Getting Started

To set up Pipedrive as a source in RudderStack, follow these steps:

1. Log into your [RudderStack dashboard](https://app.rudderstack.com/).
2. Go to **Sources** > **New source** > **Cloud Extract** and select **Pipedrive V2** from the list of sources.
3. Assign a name to your source and click **Continue**.

### Connection settings

To set up Pipedrive as a Cloud Extract source, configure the following settings:

<img src="../../assets/cloud-extract-sources/pipedrive-connection-settings.png" alt="Pipedrive credentials" />

> - **API Token**: Enter your Pipedrive API token. Refer to the [Pipedrive documentation](https://pipedrive.readme.io/docs/how-to-find-the-api-token) for more information on obtaining the API token.

Reviewer 2: A small change we need to reorder the values. 
![Screenshot 2023-02-09 at 6 02 04 PM](https://user-images.githubusercontent.com/1304376/217868533-444d02a2-4169-4236-8ef9-0d5e2163f68a.png)

### Destination settings

The following settings specify how RudderStack sends the data ingested from Pipedrive to the connected warehouse destination:

- **Table prefix**: RudderStack uses this prefix to create a table in your data warehouse and loads all your Pipedrive data into it.
- **Schedule Settings**: RudderStack gives you three options to ingest the data from Pipedrive:
    - **Basic**: Runs the syncs at the specified time interval. 
    - **CRON**: Runs the syncs based on the user-defined CRON expression.
    - **Manual**: You are required to run the syncs manually.

<div class="infoBlock">
For more information on the schedule types, refer to the <Link to="/sources/extract/common-settings/">Common Settings</Link> guide.
</div>

### Selecting the data to import

You can choose the Pipedrive data you want to ingest by selecting the required resources:

<img src="../../assets/cloud-extract-sources/pipedive-v2-resources.png" alt="Selecting the data to import" />

The below table mentions the syncs supported by these resources from Pipedrive to your warehouse destination where `id` is a common primary key for all:  

| Resource | Sync type | 
| :---| :--- | 

> |`users`| Incremental | 

Reviewer: Do we have any details on the associated endpoints?

Author: @am6010 If you can add the endpoints in the spec, I'll add those to the docs

Reviewer 2: I have updated the doc 
https://www.notion.so/rudderstacks/Source-Pipedrive-ade0866afec544d299f9bfe04d7951d3
